stl/inc/xstring

```diff
@@ -3517,8 +3600,8 @@ public:
         _Mypair._Myval2._Check_offset(_Off);
         const size_type _Old_size = _Mypair._Myval2._Mysize;
 
-        // checking for overlapping ranges is technically UB (considering string literals), so just always reallocate
-        // and copy to the new buffer if constant evaluated
+        // checking for overlapping ranges is technically UB (considering string literals), so just always

```

Comment:
An isolated one-word comment reflow like this is generally the result of saving (and formatting-on-save)) with an unmatched `{` earlier in the file. The unmatched brace adds an indent for the entire remainder of the file, causing lines near the maximum length to wrap; when the unmatched brace is corrected and the indent removed, non-comments normally return to the prior state but clang-format only touches over-length comment lines.

And +1 to unspecified instead of UB being the reason we can't check for overlapping in constant evaluation; I'll clean this up.